lib/simple_formatter_patch.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+ActiveSupport::LogSubscriber.colorize_logging = false
+
+class ActiveSupport::Logger::SimpleFormatter
+  def call(severity, time, progname, msg)
+    formatted_severity = severity[0]
+    formatted_time = time.strftime("%Y-%m-%d %H:%M:%S.") << time.usec.to_s[0..2].rjust(3)
+    caller_location = get_caller_location
+    args = format_args(msg)
+
+    "#{formatted_time} | CMEvaluator | CMEvaluator | #{formatted_severity} | #{caller_location} | #{args}\n"
+  end
+
+  def format_args(args)
+    output = if args.is_a?(Hash)
+               # Format args in key=value pair, separated by pipes
+               args.map do |key, value|
+                 "#{key}=#{value}"
+               end.join(' | ')
+             else
+               args.to_s.gsub(/\033\[1;4;32m(.*)\033\[0m/, '\1')
```

Comment:
This follows the behaviour of Superlogger:

If it receives a hash it will print out as `msg=foo`
Otherwise, it will just print out the msg converted to string.